frontend/public/components/app.jsx

```diff
@@ -202,13 +204,15 @@ class App_ extends React.PureComponent {
   }
 }
 
-initConsolePlugins(store);
+const AppWithExtensions = withTranslation()((props) => {
```

Comment:
@glekner We should utilize the second tuple element (`resolved: boolean`) of `useResolvedExtensions` hook result.

This tells us whether the `CodeRef` resolution for all matching extensions is complete or still in progress. We generally don't want to re-render unless *all* extensions have their `CodeRef` properties resolved.